nc-news-fe/src/components/CommentContent.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import CommentCard from "./CommentCard";
+
+const CommentContent = ({ comments }) => {
+  const [comment, setComment] = useState("");
+  const [showComments, setShowComments] = useState(false);
+
+  return (
+    <section>
+      <form className="d-flex flex-column align-items-center">
+        <div className="d-flex">
+          <label htmlFor="comment">
+            <input
+              type="text"
+              value={comment}
+              onChange={(e) => setComment(e.target.value)}
+            />
+          </label>
+          <button>Add</button>
+        </div>
+      </form>
+      <h2>Most Recent Comment</h2>

```

Comment:
Your label at the bottom of the page reads 'most recent comment' but it doesn't actually show a comment? This is a bit confusing as the user might expect to be seeing a comment here' but it's blank